beancount/ops/validation.py

```diff
@@ -87,7 +87,7 @@ def validate_open_close(entries, unused_options_map):
             else:
                 try:
                     open_entry = open_map[entry.account]
-                    if entry.date <= open_entry.date:
+                    if entry.date < open_entry.date:
```

Comment:
This should be a separate commit with a commit message that explain what is being changed and why. That said, I haven't checked if there is a reason for not allowing `open` and `close` directives on the same day. @blais do you remember anything about this?

Reply:
I don't remember.
Il'l review now.

Reply:
It does make sense to me that we should allow entries on the same day as the open though. Accepting.